pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1448,7 +1448,6 @@ func getCPUTopology(vmi *v1.VirtualMachineInstance) *CPUTopology {
 			sockets = vmiCPU.Sockets
 		}
 	}
-
```

Comment:
Maybe we can somehow clarify that we need this prition for backward-compatibility.

Reply:
Will do